CHANGELOG.md

```diff
@@ -50,7 +52,6 @@
 - PR #6765 Cupy fallback for __array_function__ and __array_ufunc__ for cudf.Series
 - PR #6817 Add support for scatter() on lists-of-struct columns
 - PR #6805 Implement `cudf::detail::copy_if` for `decimal32` and `decimal64`
-- PR #6847 Add a cmake find module for cuFile in JNI code
```

Comment:
This PR was merged to `0.18` but entry is in the `0.17` section, so moved this around.